index.html

```diff
@@ -116,18 +116,23 @@
 
 It's not an abitrary choice, and it's not one that was looked at lightly.
 - Globally accepted standard
-- Versatility of being distributed
+- Versatility of being distributed (DVCS)
 - Free & Open-Source!!
 - Enabling better development practices
 - Critical in taking further steps towards modern application development.
+- And finally, even Microsoft made the jump!!!
 
 There are tools to help the transition!
 
 ---
 
 ## About Git
 
----
+By far, the most widely used modern version control system in the world today is Git. Git is a mature, actively maintained open source project originally developed in 2005 by Linus Torvalds, the famous creator of the Linux operating system kernel.  
```

Comment:
separating sentences on new lines (will format to 1 paragraph)

```suggestion
By far, the most widely used modern version control system in the world today is Git. 
Git is a mature, actively maintained open source project originally developed in 2005 by Linus Torvalds, the famous creator of the Linux operating system kernel.  
```